router/batchrouter/batchrouter.go

```diff
@@ -1797,7 +1797,7 @@ func (worker *workerT) workerProcess() {
 			rruntime.Go(func() {
 				switch {
 				case misc.ContainsString(objectStorageDestinations, brt.destType):
-					destUploadStat := stats.DefaultStats.NewStat(fmt.Sprintf(`batch_router.%s_dest_upload_time`, brt.destType), stats.TimerType)
+					destUploadStat := stats.NewTaggedStat(`batch_router.dest_upload_time`, stats.TimerType, map[string]string{"destType": brt.destType})
```

Comment:
We need to have both measurements (old & new) and only remove the old measurement as soon as operator changes have been deployed. these measurement are already used in grafana dashboards

Reply:
Which panel is using these? I don't see any on hosted grafana

Reply:
https://github.com/rudderlabs/rudderstack-operator/blob/85a7eaf55b07db21b3faa7001fb7f586bc9039aa/helm-charts/renders/onprem/rudderstack-monitor/charts/grafana/templates/ruddermetric-dashboards-json-configmap.yaml#L4693

Reply:
the panel's name is **Batch Router destination (S3, MinIO, etc.)**